concent_api/core/message_handlers.py

```diff
@@ -444,75 +459,143 @@ def handle_send_force_subtask_results_response(request, client_message):
     return HttpResponse("", status = 202)
 
 
+def sum_payments(payments: List[Union[ForcedPaymentEvent, BatchTransferEvent]]):
+    assert isinstance(payments, list)
+
+    return sum([item.amount for item in payments])
+
+
+def sum_subtask_price(subtask_results_accepted_list: list):
+    assert isinstance(subtask_results_accepted_list, list)
+
+    return sum([subtask_results_accepted.task_to_compute.price for subtask_results_accepted in subtask_results_accepted_list])
+
+
+def sum_amount_price_for_provider(
+    list_of_forced_payments:        List[ForcedPaymentEvent],
+    list_of_payments:               List[BatchTransferEvent],
```

Comment:
I think this should be `Union` of `List[BatchTransferEvent]` and `List[dict]` because the second one is returned by `mock` backend.

Reply:
@rwrzesien do we still need `mock` backend? Even if so, I am not sure if we should change production code types just to match our testing solution...

Reply:
@dybi We should keep it if anyone ever would like to run Concent without accessing existing blockchain. It is not only testing solution, it is a possibility to run Concent in this way.

Reply:
I think we should replace the mock backend with a null backend - on that just returns empty values, to be used in cases where we don't want to connect to the blockchain.

I don't think a mock backend with hard-coded values is a good idea outside of tests.

Reply:
Done